src/test/java/racingcar/domain/ApplicationTest.java

```diff
@@ -1,62 +1,57 @@
 package racingcar.domain;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertAll;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static racingcar.common.info.RacingGameErrorInfo.*;
 
-import java.lang.reflect.Method;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import org.assertj.core.api.AbstractBooleanAssert;
-import org.assertj.core.api.Assertions;
+import java.util.stream.Collectors;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import racingcar.common.UserInput;
-import racingcar.common.info.RacingGameErrorInfo;
-import racingcar.common.info.RacingGameInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import racingcar.controller.dto.InputDTO;
+import racingcar.controller.dto.OutputDTO;
+import racingcar.model.domain.FixNumberBehavior;
+import racingcar.model.domain.Racing;
+import racingcar.model.domain.RandomNumberBehavior;
+import racingcar.model.domain.Victory;
+import racingcar.model.entity.Car;
+import racingcar.model.utils.common.Validation;
+import racingcar.view.RacingCarUserConsole;
 
 public class ApplicationTest {
 
-    private UserInput userInput;
+    private RacingCarUserConsole userConsole;
+    private static final String COMMAS = ",";
+    private InputDTO inputDTO;
 
     @DisplayName("객체 초기화")
     @BeforeEach
     void init() {
-        userInput = new UserInput();
+        inputDTO = new InputDTO(Arrays.asList("a", "aa", "aaa"), "CarNameIsLong", 10);
     }
 
-    @DisplayName("유저_입력_글자길이_테스트")
+    @DisplayName("유저_입력_글자길이_및_입력_중복_테스트")
     @Test
-    void userInputLengthTest() {
-        userInput.setCarNameInput("CarNameIsLong");
-        assertTrue(userInput.parsingCarName() == true);
-    }
-
-    @DisplayName("유저_입력_중복_테스트")
-    @Test
-    void userInputDuplicationTest() {
-        userInput.setCarNameInput("Benz,Kia,Kia");
-        assertThat(userInput.parsingCarName() == true);
+    void userInputLengthDuplicationTest() {
+        assertTrue(Validation.parsingCarName(inputDTO.getCarNameInput()) == true);
```

Comment:
boolean비교를 넣지 않아도 될 것으로 보여요~!

```suggestion
        assertTrue(Validation.parsingCarName(inputDTO.getCarNameInput()));
```

Reply:
반영하였습니다. :)
감사합니다.